kategory-core/src/main/kotlin/kategory/arrow/FunctionK.kt

```diff
@@ -1,6 +1,6 @@
 package kategory
 
-interface FunctionK<in F, out G> {
+interface FunctionK<F, G> {
```

Comment:
Why have they lost their variance?

Reply:
Inferred types with the type-literal in java reflection not liking contravariant positions. Also these are usually mapped to root ADTs and typeclasses which are just invariant positions AFAIK.